### PR TITLE
feat(svg-icon): build svg-icons package with lit (ionicons)

### PR DESCRIPTION
**Pull request by MM25Zamanian**

### CODE REVIEW

File: package/svg-icon/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@theone/util",
```
Comment (MM25Zamanian) [inline package/svg-icon/package.json:2]: Change name

File: package/svg-icon/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "version": "0.0.0",
+  "description": "Typescript/Javascript Icon Pack Lit",
+  "main": "index.js",
```
Comment (MM25Zamanian) [inline package/svg-icon/package.json:5]: Create index file

**Review by MM25Zamanian — COMMENTED** [review]
@asmanp2012 
Ok??

**Comment by MM25Zamanian** [comment]
Please review again !
@asmanp2012

**Review by asmanp2012 — APPROVED** [review]
Every thing is ok

**Pull request merged**